docs/arch_and_scale/index.rst

```diff
@@ -0,0 +1,33 @@
+.. _RabbitMQ Clustering Guide: https://www.rabbitmq.com/clustering.html
+
+Architecture and Scaling
+========================
+
+Pulp's architecture has five components of it. Each of these can be horizontally scaled
```

Comment:
Minor point: "has five components." or "has five components to it" I think sounds better than of it here.

Reply:
I agree, fixed in next push. ty.